cpptests/t_client.cpp

```diff
@@ -1,20 +1,22 @@
 #include <gtest/gtest.h>
 #include <cstdlib>
 #include <string>
+#include <cassert>
+#include <climits>
+#include <netdb.h>
+
 #include "hiredis.h"
 #include "common.h"
 
-
-using namespace hiredis;
+namespace hiredis {
```

Comment:
I'm not a fan of using scopes in source files

Reply:
Why?